Tools/gazebo_sitl_multiple_run.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+# run multiple instances of the 'px4' binary, but w/o starting the simulator.
```

Comment:
Is that part of the comment accurate? Seems to me like it is starting `gzserver`, right?

Reply:
You are right. I will update the comment

Reply:
Ok! I'll try to run it as soon as that's fixed, so that I can review the final state :blush:

Reply:
@JonasVautherin Done!